content/pages/mua.md

```diff
@@ -84,7 +84,13 @@ it can be corrected.
 
 ## Auditing and Logging ##
 
-The WiCS Systems Committee keeps records ("logs") on our machines to
+The WiCS Systems Committee keeps records ("logs") on our machines to:
+
+1. Have diagnostic information to refer to when hardware and software issues arise
```

Comment:
Can we format this like the constitution? With semicolons at the end until the last line with a period!

Also I'd change this line to
"Collect and store diagnostic information to aid in debugging hardware and software issues;"